mountd/rpc.c

```diff
@@ -325,7 +325,9 @@ int ndr_write_union_##name(struct ksmbd_dcerpc *dce, type value)	\
 	int ret;							\
 									\
 	ret = ndr_write_##name(dce, value);				\
-	ret |= ndr_write_##name(dce, value);				\
+	if (ret)							\
+		return -ENOMEM;						\
```

Comment:
any reason why you return -ENOMEM here instead of ret variable ?

Reply:
Nope, no reason